p2p/dialer.go

```diff
@@ -188,7 +188,8 @@ func (m *dialer) startConfigExchange(config *p2pConnectConfig, ctx context.Conte
 	}
 
 	// Parse provider response with public key and encrypted and signed connection config.
-	exchangeMsgReplySignedMsg, err := unpackSignedMsg(m.verifier, exchangeMsgBrokerReply)
+	// TODO: to verifier factory
```

Comment:
What does this TODO mean? It's not clear to me. Could you explain it more clear?

Reply:
Sorry, forgot to remove. Now it's gone. Was a mark for me I accidentally left.